app/AppKernel.php

```diff
@@ -30,6 +30,7 @@ public function registerBundles()
             new SumoCoders\FrameworkUserBundle\SumoCodersFrameworkUserBundle(),
             new SumoCoders\FrameworkSearchBundle\SumoCodersFrameworkSearchBundle(),
             new SumoCoders\FrameworkErrorBundle\SumoCodersFrameworkErrorBundle(),
+            new SumoCoders\FrameworkExampleBundle\SumoCodersFrameworkExampleBundle(),
```

Comment:
ik zou dat in `if (in_array($this->getEnvironment(), array('dev', 'test'))) {` zetten. (en dan ook routes in routing_dev.yml.

In prod wil je nooit die example bundle hebben.